Kconfig

```
@@ -447,6 +447,23 @@ config DFU_PID
 	hex "PID to use for DFU image"
 
 endif # DFU_BINARY
+
+config OUTPUT_STRIP_PATHS
+	bool "Strip absolute paths from binaries"
+	default y
+	depends on ARCH_TOOLCHAIN_GNU
+	---help---
+	  Cmake build provide absolute paths to compile files. If __FILE__
+	  macros are used in the source code(ASSERT), the binary will be occupied
+	  by many invalid paths.
+	  If the compiler support preprocessing files and expand the "__FILE__"
+	  and "__BASE_FILE__" macros as if the files resided in directory new instead.
```

```suggestion
	  Cmake build provide absolute paths to compile files. If __FILE__
	  macros are used in the source code(ASSERT), the binary will contain
	  many invalid paths.
```


This sentence does not make sense, so I do know what to suggest for "new"

 If the compiler support preprocessing files and expand the "__FILE__"
	  and "__BASE_FILE__" macros as if the files resided in `directory new` instead.

---

Done